src/shared/ajv.ts

```diff
@@ -1,27 +1,12 @@
-import Ajv from "ajv";
-import addFormats from "ajv-formats";
-import openAttestationSchemav2 from "../2.0/schema/schema.json";
-import openAttestationSchemav3 from "../3.0/schema/schema.json";
-import { CurrentOptions } from "ajv/dist/core";
+import { AnyValidateFunction } from "ajv/dist/core.js";
+import strictCompiledSchema from "../__generated__/compiled_schema_strict.js";
+import nonStrictCompiledSchema from "../__generated__/compiled_schema_non_strict.js";
 
-const defaultTransform = (schema: Record<string, any>) => schema;
-export const buildAjv = (
-  options: CurrentOptions & { transform: (schema: Record<string, any>) => Record<string, any> } = {
-    transform: defaultTransform,
-  }
-): Ajv => {
-  const { transform, ...ajvOptions } = options;
-  const ajv = new Ajv({ allErrors: true, allowUnionTypes: true, ...ajvOptions });
-  addFormats(ajv);
-  ajv.addKeyword("deprecationMessage");
-  ajv.compile(transform(openAttestationSchemav2));
-  ajv.compile(transform(openAttestationSchemav3));
-  return ajv;
-};
-
-const localAjv = buildAjv();
-export const getSchema = (key: string, ajv = localAjv) => {
-  const schema = ajv.getSchema(key);
+export const getSchema = (key: string, mode = "strict"): any => {
+  const schema =
+    mode === "strict"
+      ? strictCompiledSchema[key as keyof typeof strictCompiledSchema]
+      : nonStrictCompiledSchema[key as keyof typeof nonStrictCompiledSchema];
   if (!schema) throw new Error(`Could not find ${key} schema`);
-  return schema;
+  return schema as AnyValidateFunction;
```

Comment:
What is the difference between `AnyValidateFunction` vs `ValidateFunction`?

Reply:
Oh `AnyValidateFunction` caters for `async` validations. Just realised we dont use it in our validation, Ill revert to using `ValidateFunction` like before!